src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -0,0 +1,5 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+
+<ti:ajaxRemindParticularStudentsCheckboxes />
```

Comment:
Having an extra tag is pretty useless when your jsp is doing nothing more than calling it.